config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -1,8 +1,17 @@
 [docker_cli/systemd]
-docker_timeout = 300
+subsubtests = systemd_pull, systemd_build , systemd_run
+
 #: Systemd unit file
+sysd_unit_file = docker-pull.service
+#: image name to pull, build, run
+image_name = rhel7:latest
```

Comment:
I like using `busybox`. It's always there, and it's very small (1MB) so it's a quick pull.